test/bdd/steps/api/search-assertion.js

```diff
@@ -0,0 +1,76 @@
+const { When, Then, Given } = require('@cucumber/cucumber');
+const { expect, assert } = require('chai');
+const utilities = require('../../../utilities/utilities');
+let handlerId, searchResult, searchData, parsedKeywords, publishData;
+
+Given(/^I call search request on node (\d+) with ([^"]*) for the keywords:*$/, { timeout: 120000 }, async function (node, resultType, keywords) {
+    resultType = 'assertions';
+    parsedKeywords = utilities.unpackRawTableToArray(keywords);
+
+    const result = await this.state.nodes[node - 1].client
+        .search(resultType, parsedKeywords[0]).catch((error) => {
+            assert.fail(`Error while trying to search assertion. ${error}`);
+        });
+
+    expect(result.status).to.be.eql(202);
+    expect(result.statusText).to.be.eql('Accepted');
+    expect(result.data.handler_id).to.be.a('string');
+    handlerId = result.data.handler_id;
+
+    if (handlerId === undefined) {
```

Comment:
We don't need this step, since expect on line 17 will fail the handlerId is undefined